restoration.txt

```diff
@@ -3,10 +3,11 @@ Name = Restoration
 Version = 3.2.13
 Description = Restoration is a freeware, portable app that can restore files accidentally deleted from the recycle bin.
 Category = 12
-URLDownload = https://files1.majorgeeks.com/10afebdbffcd4742c81a3cb0f6ce4092156b4375/drives/Restoration.zip
+URLDownload = https://web.archive.org/web/2024if_/https://files1.majorgeeks.com/acd9affbbee5e4c28a3a457dcf138bb1259439e4/drives/Restoration.zip
```

Comment:
Different "hash" in the URL, but the resulting file's SHA1 and size (below) are the same?

Reply:
It seems to be a weird old clumsy software/abandonware, and majorgeeks it's not an official source, neither the softpedia, neither the one I upload here. It's hard to find the original source. I will try to find that one.
https://web.archive.org/web/20210328174143/www3.telus.net/mikebike/Restoration.zip